charts/base/templates/api-gateway-base/configmap.yaml

```diff
@@ -23,11 +22,14 @@ data:
     MGMT_BACKEND_PUBLICPORT={{ template "base.mgmtBackend.publicPort" . }}
 
     # influxdb
-    INFLUXDB_ENABLED={{ .Values.tags.observability }}
+    INFLUXDB_ENABLED={{ .Values.tags.prometheusStack }}
```

Comment:
For Instill Cloud, since we have the prometheus stack always deployed by `kube-prometheus-stack`, I suppose `.Values.tags.prometheusStack` will be set `false` here? But if it is set `false`, the KrakenD won't enable InfluxDB metrics.